feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -754,6 +754,18 @@ func TestP4rtNodeID(t *testing.T) {
 	}
 }
 
+func fetchInAndOutPkts(t *testing.T, dut *ondatra.DUTDevice, dp1, dp2 *ondatra.Port, isInterfaceCountersFromContainer bool) (uint64, uint64) {
```

Comment:
Can you remove the isInterfaceCountersFromContainer parameter and just do the lookup in this method? It's a constant time lookup.